draft-wang-ppm-differential-privacy.md

```diff
@@ -324,16 +324,11 @@ in DAP. It does so in the following stages:
    systems, including algorithms for sampling from discrete Laplace and
    Gaussian distributions.
 
-1. {{policies}} defines DP policies and specifies concrete policies for
-   endowing VDAFs in {{!VDAF}} with DP. [TODO: And other drafts, once they
-   appear.] [CP: This API may not be compatible with all DP polices we might
-   want to implement.]
+1. {{policies}} defines DP policies, their composition with VDAFs, and their
```

Comment:
Feel free to take it or not, the goal is to connect with the previous bullet point:

```suggestion
1. {{policies}} defines DP policies that are implemented with DP mechanisms, their composition with VDAFs, and their
```